.github/workflows/build.yml

```diff
@@ -17,36 +17,35 @@ jobs:
 
       - name: Set Environment Variables
         run: |
+               echo >> $GITHUB_ENV POSTGRES_USER='prime'
+               echo >> $GITHUB_ENV POSTGRES_PASSWORD='changeIT!'
+               
                if [ "$GITHUB_BASE_REF" == "production" ]
                then
                    echo "Building for the production environment."
                    echo >> $GITHUB_ENV ACR_REPO=pdhprodcontainerregistry.azurecr.io
                    echo >> $GITHUB_ENV PREFIX=pdhprod
-                   echo >> $GITHUB_ENV POSTGRESQL_USER=${{ secrets.POSTGRESQL_PROD_USER }}
-                   echo >> $GITHUB_ENV POSTGRESQL_PWD=${{ secrets.POSTGRESQL_PROD_PWD }}
                else
                    echo "Building for the test environment."
                    echo >> $GITHUB_ENV ACR_REPO=pdhtestcontainerregistry.azurecr.io
                    echo >> $GITHUB_ENV PREFIX=pdhtest
-                   echo >> $GITHUB_ENV POSTGRESQL_USER=${{ secrets.POSTGRESQL_TEST_USER }}
-                   echo >> $GITHUB_ENV POSTGRESQL_PWD=${{ secrets.POSTGRESQL_TEST_PWD }}
```

Comment:
AFAICT, the DB-related environment variables never get incorporated into the build artifact, so this seems like a safe change, and simplifies things. We could also just get rid of these environment variables entirely, and hard-code the values in the “Setup PostgreSQL” step like we do with the database name.

Reply:
@Mr0grog - would you be able to revert this change?  Since we are not going to perform builds of forked PRs, we are going to keep these as secrets.

Reply:
Yep, will do.

FWIW, though, I *do* still think keeping them as plaintext would be better (as noted here: https://github.com/CDCgov/prime-data-hub/pull/271#issuecomment-759657862). These passwords are in plaintext elsewhere in the codebase, and you are already restricting who this workflow runs for via other means. You’d have to change this back to plaintext in the future if you ever loosened those restrictions (e.g. using GitHub’s repo moderation settings or creating a blocklist instead of basing it on the PR branch’s repo).